.github/workflows/docs.yml

```diff
@@ -25,7 +25,7 @@ jobs:
             max_attempts: 3
             command: |
               composer update
-              composer require --dev --with-dependencies phpdocumentor/reflection:^4.0
+              composer require --dev --with-all-dependencies phpdocumentor/reflection:^4.0
```

Comment:
We could just run `composer require --dev phpdocumentor/reflection:^4.0`, and `composer update` will be implicitly run because there is no `composer.lock` file present...

Reply:
Ok, let's give it a shot - updated!